HISTORY.md

```diff
@@ -1,5 +1,7 @@
 ### mlpack ?.?.?
 ###### ????-??-??
+  * Added Extra Trees Algorithm (#2883).
```

Comment:
Can you mention the `tree::ExtraTrees` class in this log entry?  (I think that's the namespace... correct me if I'm wrong...)  That will be helpful for users to find it. :+1: